mode/examples/Topics/Image Processing/Blending/Blending.pyde

```diff
@@ -21,17 +18,17 @@ modes = ((REPLACE, "REPLACE"),
          (MULTIPLY, "MULTIPLY"),
          (SCREEN, "SCREEN"),
          (REPLACE, "REPLACE"))
+img1 = loadImage("layer1.jpg")
```

Comment:
Wait, that sounds like a bug. What exceptions?

On Mon, Feb 9, 2015 at 3:12 PM, tildebyte <notifications@github.com> wrote:

> In mode/examples/Topics/Image Processing/Blending/Blending.pyde
> <https://github.com/jdf/processing.py/pull/135#discussion_r24360955>:
>
> > @@ -21,17 +18,17 @@ modes = ((REPLACE, "REPLACE"),
> >           (MULTIPLY, "MULTIPLY"),
> >           (SCREEN, "SCREEN"),
> >           (REPLACE, "REPLACE"))
> > +img1 = loadImage("layer1.jpg")
>
> Yep. In fact, check the message from 45c3c01
> <https://github.com/jdf/processing.py/commit/45c3c01599b512411c4940861c45bd31bf8da12f>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdf/processing.py/pull/135/files#r24360955>.
>



-- 
Jonathan Feinberg  jdf@pobox.com  http://MrFeinberg.com/